data/src/main/java/com/architect/coders/mu8/data/service/MarvelServiceAPI.kt

```diff
@@ -45,4 +47,22 @@ interface MarvelServiceAPI {
         @Query(OFFSET) offset: Int,
         @Query(LIMIT) limit: Int
     ): Response<BaseResponse<EventsResponse>>
+
+    @GET("/v1/public/characters/{characterId}")
+    suspend fun findById(
```

Comment:
find what by id? Please give a better name for this function.

Reply:
findCharacterById